hopsworks-admin/src/main/java/io/hops/hopsworks/admin/user/account/ResetPassword.java

```diff
@@ -503,7 +503,7 @@ public String logout() {
       req.logout();
       session.invalidate();
       usersController.setOnline(people.getUid(), -1);
-      context.getExternalContext().redirect("/hopsworks/#!/home");
+      context.getExternalContext().redirect("/aegis-web/#!/home");
```

Comment:
shouldn't this be `/aegis/#!/home`? what is aegis-web?

Reply:
changed